index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
be careful to avoid introducing changes like this. Nothing changed!! Why is this getting shown.